models/payment_rest.go

```diff
@@ -70,5 +70,6 @@ type CostResourceRest struct {
 	ClassOfPayment          []string          `json:"class_of_payment"          validate:"required"`
 	Description             string            `json:"description"               validate:"required"`
 	DescriptionIdentifier   string            `json:"description_identifier"    validate:"required"`
+	ProductType 			string            `json:"product_type"              validate:"required"`
```

Comment:
Indentation: please enable `gofmt` and `goimports` to automatically run in your editor, against edited files.